### DISCUSSION TITLE
Problems with gotek on Amstrad PC5286.

### DISCUSSION THREAD

**Discussion by Starvak** [keirf/flashfloppy#616] [General]
Hello. Restoring an Amstrad PC5286, I had to replace the old floppy drive. In any case, installing the gotek is ideal for this small PC. I had to buy an adapter from 26 pins to 34 pins, and connecting the gotek with the jumper JC and DS1.
After the first boot, everything seemed to be going well. I insert the usb and I can read and write like a normal disk. But when selecting another disk n2 (with the rotary control or pressing the button indistinctly), and doing a "dir" the content of the previous diskette continues to appear, and it does not read the content of the new one.
What am I doing wrong?
The gotek is model sfrc922, with firmware FF 3.28.
Looking for a possible solution, I have found that by pressing "crtl+c" and then "dir", it does read the content of the new disk n2.

**Comment by keirf** [comment]
You probably need a jumper at position JC (if your Gotek has that jumper location) or else `interface=ibmpc` in the FF.CFG config file.

**Reply by Starvak** [reply]
Yes, I have both the config and the jumper active as I mentioned.
I have tried with a normal floppy drive sony mpf-131, and the same thing happens to me. I think I have to look for more information about the floppy drive ic that mounts the PC4286 and about the original citizen U1DA-38A floppy drive that he came from  factory.

**Comment by keirf** [comment]
Have you also tried without the JC jumper?

**Reply by Starvak** [reply]
I have always tried with JC connected.
I'm going to try it now without the JC, as you say.

**Reply by Starvak** [reply]
---------------------------
I edit and inform.
There is no change without the JC jumper. I understand the xxx configuration is doing its job.

**Comment by Starvak** [comment]
I have tried with a real sony mpf920 floppy drive. Everything happens the same; I insert disk n1 and do "dir" it shows me the content. I remove n1 and put n2 disk, I do "dir" and the content of disk n1 appears and not that of n2.
If before inserting disk #2, I hit "ctrl+c" I get a little "break" command. after that, I put disk n2 and do dir, then it does show the content of disk n2.
Therefore, it seems that it is not a problem with gotek or the FF firmware.

I'm going to physically disassemble and inspect the motherboard.

**Reply by Starvak** [reply]
Nothing strange on the motherboard equipment. I thought I would find some 82077AA chip, but instead, there is a Gal programmed and some hex inverters... well, I'm actually a lawyer, I don't understand further, I'm like a parrot, but I don't I understand these issues thoroughly.

At this point, I am pretty sure that it´s a problem between the 26 pin cable from the motherboard and the Chinese to 34 pin converter. I have already used this converter before on another pc successfully, so the converter should be fine.

It must be an amstrad proprietary pinout on the pc5286 model, which works fine only with floppy drives like the factory mounted citizen U1DA-38A. Something affects the diskchange signal, maybe it is not properly connected because it is on a different pin.

I will focus on looking for the amstrad pc5286 or U1DA-38A service manuals to check the pinout signals.
If anyone finds any, **please let me know**; I've been searching for a while with no results.

Many thanks to everyone who read the thread, and to Keirf especially for the FF firmware. Retro users have a new life

I put here generic information of the pinout 26 to 34 pins
![26pins](https://user-images.githubusercontent.com/32796045/158016631-e6641489-e3d3-4e40-bb4d-d311e131f171.jpg)
.
